includes/uploader.js

```diff
@@ -40,9 +40,72 @@ const expectedColumnNames = [
 // Initialise global variables to store row values
 var rowValues = [];
 
+// Check table and initialize session variable to handle file upload properly
+function checkPreviousSessions() {
+    // Get the table element by its ID
+    let table = document.getElementById('tableData');
+
+    // Check if the table element exists
+    if (!table) {
+        console.error('Table with id "tableData" not found.');
+        return;

```

Comment:
If we keep the `return sessionState` below, I would make it explicit that we're returning `false` here